xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsWatchUIX.xml

```diff
@@ -0,0 +1,656 @@
+  #set ($watchIcon = $iconWhenBlocked)
+  #set ($watchText = $services.localization.render('notifications.watch.button.status.blocked'))
+#else
+  #set ($watchIcon = $iconWhenCustom)
+  #set ($watchText = $services.localization.render('notifications.watch.button.status.custom'))
+#end
+## TODO: better way for that?
+#set ($isTerminal = ($doc.getDocumentReference().name != 'WebHome'))
+#macro (_displayOption $attributeName $optionValue $translationSuffix $iconName)
+  &lt;div class="panel panel-default"&gt;
+    &lt;div class="panel-radio"&gt;
+      &lt;input type="radio" name="watch-option" id="option-$attributeName" value="$optionValue" aria-labelledby="title-$attributeName" /&gt;
+    &lt;/div&gt;
+    &lt;div class="panel-heading" role="tab" id="heading-$attributeName"&gt;
+      &lt;a role="button" data-toggle="collapse" data-parent="#watch-options-accordion" href="#xhint-$attributeName" aria-controls="xhint-$attributeName"&gt;
+        &lt;h4 class="panel-title" id="title-$attributeName"&gt;
+          $services.icon.renderHTML("$iconName") $services.localization.render("notifications.watch.modal.option.${translationSuffix}.title")
+        &lt;/h4&gt;
+      &lt;/a&gt;
+    &lt;/div&gt;&lt;!-- end panel heading --&gt;
+    &lt;div id="xhint-$attributeName" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading-$attributeName"&gt;
+      &lt;div class="panel-body xHint"&gt;
+        $services.localization.render("notifications.watch.modal.option.${translationSuffix}.hint")
+      &lt;/div&gt;
+    &lt;/div&gt;&lt;!-- end collapsed xhint --&gt;
+  &lt;/div&gt;&lt;!-- end panel --&gt;
+#end
+#set ($watchPageOption = "#_displayOption('watch-page','watchPage','watchpage','page')")
+#set ($watchSpaceOption = "#_displayOption('watch-space','watchSpace','watchspace','chart-organisation')")
+#set ($watchWikiOption = "#_displayOption('watch-wiki','watchWiki','watchwiki','world')")
+#set ($blockPageOption = "#_displayOption('block-page','blockPage','blockpage','page')")
+#set ($blockSpaceOption = "#_displayOption('block-space','blockSpace','blockspace','chart-organisation')")
+#set ($unwatchPageOption = "#_displayOption('unwatch-page','unwatchPage','unwatchpage','page')")
+#set ($unwatchSpaceOption = "#_displayOption('unwatch-space','unwatchSpace','unwatchspace','chart-organisation')")
+#set ($unwatchWikiOption = "#_displayOption('unwatch-wiki','unwatchWiki','unwatchwiki','world')")
+#set ($unblockPageOption = "#_displayOption('unblock-page','unblockPage','unblockpage','page')")
+#set ($unblockSpaceOption = "#_displayOption('unblock-space','unblockSpace','unblockspace','chart-organisation')")
+#set ($unblockWikiOption = "#_displayOption('unblock-wiki','unblockWiki','unblockwiki','world')")
+{{html clean='false'}}
+&lt;div class="btn-group" id="watchButton"&gt;
+  &lt;a href="#" role="button" title="$escapetool.xml($watchText)" class="btn btn-default" data-toggle="modal" data-target="#watchModal"&gt;
```

Comment:
Buttons should use a button element unless there is a problem with that.

Reply:
I think I reused some existing code for that one. I'll check.